chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -594,6 +594,8 @@ zotero.preferences.advanced.debug.error = An error occurred sending debug output
 
 dragAndDrop.existingFiles = The following files already existed in the destination directory and were not copied:
 dragAndDrop.filesNotFound = The following files were not found and could not be copied:
+dragAndDrop.invalidLink.title = File Not Found
```

Comment:
We have a "File Not Found" already. Probably should just make a general.fileNotFound.